commands/api.go

```diff
@@ -292,6 +293,25 @@ func apiCommand(cmd *Command, args *Args) {
 		if requestLoop && !parseJSON {
 			fmt.Fprintf(out, "\n")
 		}
+
+		if requestLoop {
+			var rateLimitLeft = -1
+			var rateLimitResetMs = -1
+			var atoiErr error
+			if xRateLimitLeft := response.Header.Get("X-Ratelimit-Remaining"); len(xRateLimitLeft) != 0 {
+				rateLimitLeft, atoiErr = strconv.Atoi(xRateLimitLeft)
+				utils.Check(atoiErr)
+			}
+			if xRateLimitReset := response.Header.Get("X-Ratelimit-Reset"); len(xRateLimitReset) != 0 {
+				rateLimitResetMs, atoiErr = strconv.Atoi(xRateLimitReset)
+				utils.Check(atoiErr)
+			}
+			if rateLimitLeft != -1 && rateLimitResetMs != -1 && rateLimitLeft <= 1 {
```

Comment:
If `rateLimitLeft == 1`, doesn't that mean that there is 1 more request available during this rate limit window, and that we could take advantage of that and let the request fire rather than waiting until reset? Or am I misunderstanding this condition